src/basic/os-util.h

```diff
@@ -23,8 +23,11 @@ static inline int fopen_os_release(const char *root, char **ret_path, FILE **ret
         return fopen_extension_release(root, NULL, ret_path, ret_file);
 }
 
-int parse_extension_release(const char *root, const char *extension, ...) _sentinel_;
-int parse_os_release(const char *root, ...) _sentinel_;
+int _parse_extension_release(const char *root, const char *extension, ...) _sentinel_;
+int _parse_os_release(const char *root, ...) _sentinel_;
```

Comment:
so far we called these functions `xyz_internal`  or so

Reply:
We had this discussion before... see 138f49e452. "_" is just as clear, but is much shorter.